JuanOliveros/MISOAppMoviles
# Feature/tests hu01

**HU01 Tests**

- Access as guest
- See Albums options
- See Albums list
- Logout from main screen
- logout from menu

**HU01 changes**
